manuscript/draft.Rmd

```diff
@@ -1,47 +1,58 @@
+Forecasting the future state of ecological systems is increasingly important for
+planning, management, and testing the performance of ecological models
+[@clark2001, @dietz2017]. One area of particular interest for forecasting is
+biodiversity since it is important for ecosystem function, central to
+conservation planning, and expected to be influenced by anthropogenic global
+change. Anticipating potential changes in biodiversity is crucial for large
+scale management and conservation, and for addressing debates regarding whether
+local scale diversity is declining and may continue to decline in response to
+anthropogenic influences [@dornelas2014].
+
+Previous efforts to predict future patterns of species richness, and diversity
+more generally, have focused primarily on building species distributions models
+to quantify the spatial relationships between the occurrence of individual
+species and the environment. Forecasts for future environmental conditions are
+then used to predict where each species will occur in the future and the
+probabilities of occurrence are summed across species to predict future species
+richness [e.g., @thomas2004]. Alternatively, models that directly relate spatial
+patterns of species richness to the environment have been developed and
+generally perform equivalently to species distribution modeling based methods
+[@algar2009, @distler2015].
+
+However, despite the emerging interest in forecasting species richness and other
+aspects of biodiversity, little is known about how effectively we can anticipate
+their dynamics. What we do know comes from a small number of hindcasting
+studies, where models are built using data on species occurrence and richness
+from the past and evaluated on their ability to predict contemporary patterns
+[e.g., @algar2009, @distler2015]. These studies are a valuable first step, but
+lack several components that are important for developing good forecasting
+models and understanding how accurately these models can predict the
+future. These components of good forecasting and evaluation (Box 1) broadly
+involve: 1) expanding the use of data to include biological and environmental
+time-series [@treddenick2016]; 2) accounting for uncertainty in observations,
+processes, model choice, and forecast evaluation [@dietz2017]; and 3) conducting
+meaningful evaluations of the forecasts by hindcasting, archiving short-term
+forecasts, and comparing forecasts to baselines to determine if the forecasts
+are more accurate than assuming the system is basically static [@Perretti2013].
+
+In this paper we attempt to forecast the species richness of breeding birds at
+hundreds of locations throughout North America while following the best
+practices in Box 1 for ecological forecasting. To do this we combine 30 years of
+time-series data on bird distributions with monthly time-series of climate data
+and satellite-based remote-sensing. We compare traditional distribution modeling
```

Comment:
This might be a good place to note the large-ish increase in average temperatures over this time period, comparable to what's expected in the next ~20 years